app/Views/admin/typeSolutions/SolutionCreate.php

```diff
@@ -1 +1,129 @@
+                        <i class="ri-arrow-left-fill label-icon align-middle rounded-pill fs-16 me-2"></i>Volver
+                    </a>
+                </div>
+                <form id="createproduct-form" autocomplete="off" class="needs-validation" novalidate method="POST">
+                    <div class="row">
+                        <div class="col-md-12">
+                            <div class="card">
+                                <div class="card-header">
+                                    <label class="form-label mb-0" for="category-title-input">
+                                        <h5 class="card-title mb-0">Titulo de la Solución de Medición</h5>
+                                    </label>
+                                </div>
+                                <div class="card-body">
+                                    <div class="hstack gap-3 align-items-start">
+                                        <input type="text" class="form-control" name="name" id="name" value="" placeholder="Ingrese el título" required>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- end col -->
+                    </div>
+                    <!-- end row -->
+
+                    <div class="row">
+                        <div class="col-md-12">
+                            <div class="card">
+                                <div class="card-header">
+                                    <h5 class="card-title mb-0">Descripción de la Solución de Medición</h5>
+                                </div>
+                                <div class="card-body">
+                                    <div class="hstack gap-3 align-items-start">
+                                        <div class="flex-grow-1">
+                                            <textarea class="form-control" rows="3" name="description"></textarea>
+                                        </div>
+                                    </div>
+                                </div>
+                                <!-- end card body -->
+                            </div>
+                            <!-- end card -->
+                        </div>
+                        <!-- end col -->
+                    </div>
+
+                    <div class="row">
+                        <div class="col-md-6">
+                            <div class="card">
+                                <div class="card-header">
+                                    <h4 class="card-title mb-0">Icono</h4>
+                                </div><!-- end card header -->
+
+                                <div class="card-body">
+                                    <input type="file" class="filepond filepond-input-multiple" name="icon" data-allow-reorder="true" data-max-file-size="3MB" data-max-files="3">
```

Comment:
```suggestion
                                    <input type="file" class="filepond" name="icon">
```
All these filepond attributes I don't think it's good to put them in the html directly, if you already have a filepond configuration object in the JS it's better to put it there. For each html attribute there is a mapping to a configuration property of the filepond instances.

and why you are using the `.filepond-input-multiple` class?

Reply:
My mistake of leaving the filepond-input-multiple thing